### ISSUE TITLE
Cannot find module '@aws-cdk/cx-api'

### ISSUE THREAD

**Issue by ethanalef** [localstack/aws-cdk-local#59]
I tried the cmd: cdklocal --version, it shows the error:

=========================================================

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@aws-cdk/cx-api'
Require stack:
- /usr/local/lib/node_modules/aws-cdk/lib/api/aws-auth/sdk-provider.js
- /usr/local/lib/node_modules/aws-cdk-local/bin/cdklocal
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk/lib/api/aws-auth/sdk-provider.js:6:15)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/aws-cdk/lib/api/aws-auth/sdk-provider.js',
    '/usr/local/lib/node_modules/aws-cdk-local/bin/cdklocal'
  ]
}

=========================================================

I tried reinstall via npm but no luck.

**Comment by ethanalef** [comment]
❯ node -e 'console.log(require("aws-cdk"))'
{
  BOOTSTRAP_VERSION_OUTPUT: [Getter],
  BOOTSTRAP_VERSION_RESOURCE: [Getter],
  BUCKET_DOMAIN_NAME_OUTPUT: [Getter],
  BUCKET_NAME_OUTPUT: [Getter],
  Bootstrapper: [Getter],
  DEFAULT_TOOLKIT_STACK_NAME: [Getter],
  Mode: [Getter],
  REPOSITORY_NAME_OUTPUT: [Getter],
  SDK: [Getter],
  SdkProvider: [Getter],
  ToolkitInfo: [Getter],
  deployStack: [Getter],
  destroyStack: [Getter],
  PluginHost: [Getter],
  cli: [Getter]
}
=========================================================
❯ npm install -g aws-cdk-local aws-cdk

changed 4 packages, and audited 5 packages in 2s

found 0 vulnerabilities
=========================================================

**Comment by ethanalef** [comment]
I then install the required package:
npm install -g @aws-cdk/cx-api
npm install -g aws-sdk
npm install -g fs-extra
npm install -g chalk

=========================================================
finally I stuck in:

❯ cdklocal --version
/usr/local/lib/node_modules/aws-cdk/lib/logging.js:5
const chalk = require("chalk");
              ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /usr/local/lib/node_modules/chalk/source/index.js from /usr/local/lib/node_modules/aws-cdk/lib/logging.js not supported.
Instead change the require of index.js in /usr/local/lib/node_modules/aws-cdk/lib/logging.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk/lib/logging.js:5:15)
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk/lib/api/aws-auth/sdk-provider.js:9:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/aws-cdk-local/bin/cdklocal:9:18) {
  code: 'ERR_REQUIRE_ESM'
}
=========================================================

**Comment by adnanaylan** [comment]
Same here. When I try cdklocal with aws-cdk@2.14.0, I get the same error. With aws-cdk@2.13.0, it's fine. It breaks our CI/CD pipeline in this morning.

**Comment by makies** [comment]
me too.

I tryed aws-cdk downgrade. It’s success and no errors.

```shell
$ npm install -g aws-cdk@2.13.0
```

```shell
cdk --version
2.13.0 (build b0b744d)

cdklocal --version
2.13.0 (build b0b744d)
```

@adnanaylan Thanks for your comment.

**Comment by vertti** [comment]
Same here.

**Comment by mikelane** [comment]
cdk version 2.14.0 is broken in the same way for me. Version 2.13.0 works, though.

**Comment by liamor** [comment]
Same with cdk version 2.15.0

**Comment by whummer** [comment]
Thanks for reporting @ethanalef @adnanaylan @makies @vertti @mikelane @liamor . We have now released a new version `aws-cdk-local@2.14.0` which should provide compatibility with versions `2.14.0` and above. Please keep us posted on how it goes.. Thanks!

**Comment by whummer** [comment]
Assuming this is fixed with the latest `cdklocal` version (2.14.0 and above) - please do let us know if the problem persists.. Thanks!
